src/main/java/com/gigacal/utils/ObjectUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.gigacal.utils;
+
+import java.util.Collection;
+import java.util.Map;
+
+public class ObjectUtil {
+
+    public static boolean isEmpty(Object o) {
```

Comment:
Is it really necessary to do it this way? Especially that you're using this method to just check if the user is not null + there's a dependency that already has it implemented - org.apache.commons.commons-lang3

Reply:
Besides, I think that all of these checks can be done with Hibernate Validator at the controller level making these utils classes unnecessary.